feat: [IDP-2780]: Added the RBAC policies for workflows and hide catalog (#6602)
harness/developer-hub: docs/internal-developer-portal/rbac/resources-roles.md

![](./static/rbac-admin-usecase.png)

### Hide Catalog Entities

> Catalog Entities can be hidden from the users, except for the owner user-group by using the tags like `hidden`, `secrets` as shown in the example below. 

Reviewer: ```suggestion
Catalog Entities can be hidden from the users, except for the owner user-group by using either one of the `hidden` or `secrets` tag as shown in the example below. 
```
